slackbot.go

```diff
@@ -0,0 +1,109 @@
+package main
+
+import (
+	"log"
+	"time"
+
+	pagerduty "github.com/PagerDuty/go-pagerduty"
+	"github.com/shomali11/slacker"
+)
+
+// Init TODO
+func Init(config *Config) {
+	log.Println("Connected!")
+	client := pagerduty.NewClient(config.PagerDuty.APIKey)
+	var opts pagerduty.GetScheduleOptions
+	if schedule, err := client.GetSchedule(config.PagerDuty.Schedule, opts); err != nil {
+		log.Fatal(err)
```

Comment:
do not use Fatal inside non-main function, return proper errors to be handled by main function

Reply:
Done.